chunkflow/chunk/base.py

```diff
@@ -378,17 +378,26 @@ def crop_margin(self, margin_size: tuple = None, output_bbox: Bbox=None):
             assert output_bbox is not None
             return self.cutout(output_bbox.to_slices())
     
-    def threshold(self, threshold: float):
-        seg = self > threshold
+    def threshold(self,
+                  threshold: float,
+                  remap_high: int = None,
+                  remap_low: int = None) -> np.ndarray:
+        # Threshold
+        seg = (self > threshold).astype(np.uint8)
+
+        # Optional remapping
+        if remap_high is not None:
+            assert remap_high >= 0 and remap_high < 256
+            seg.array[seg.array == 1] = remap_high
+        if remap_low is not None:
+            assert remap_low >= 0 and remap_low < 256
+            seg.array[seg.array == 0] = remap_low
```

Comment:
The code is problematic if someone decide to set remap_high = 0, then everything will be mapped to remap_low. It is safer to replace the condition `seg.array == 1/0` with `self >/<= threshold`.

Reply:
That's a good catch. Will modify it.